src/App.tsx

```diff
@@ -1,61 +1,121 @@
 import './App.scss';
+import { useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { ToDo } from './types/ToDo';
+import { getUserById } from './components/functions/GetUserByID';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+export const initialTodos: ToDo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
+
+function getNewToDoId(todos: ToDo[]) {
+  const MaxId = Math.max(...todos.map(todo => todo.id));
```

Comment:
```suggestion
  const maxId = Math.max(...todos.map(todo => todo.id));
```
camelCase
PascalCase only for components/types/classes